rubocop-md.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
     "source_code_uri" => "http://github.com/rubocop-hq/rubocop-md"
   }
 
-  spec.required_ruby_version = ">= 2.3.0"
+  spec.required_ruby_version = ">= 2.4.0"
```

Comment:
Could you please add a change log entry for this change?

Reply:
✅